dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4403,6 +4403,43 @@ def test_retries_on_remote_filesystem(tmpdir):
         assert layer.annotations["retries"] == 2
 
 
+def test_filesystem_option(tmpdir, engine):
+    from fsspec.implementations.local import LocalFileSystem
+
+    df = pd.DataFrame({"a": range(10)})
+    dd.from_pandas(df, npartitions=2).to_parquet(tmpdir, engine=engine)
+    fs = LocalFileSystem()
+    fs._myfs = True
+    ddf = dd.read_parquet(
+        str(tmpdir),
+        engine=engine,
+        filesystem=fs,
+    )
+    layer_fs = next(iter(ddf.dask.layers.values())).io_func.fs
+    assert layer_fs._myfs
```

Comment:
I think the intent here is to make sure the same filesystem object is included in the layer. Instead of adding a special `._myfs` attribute, would `assert layer_fs is fs` test the same thing? If so, that seems a bit cleaner

Reply:
good point